### ISSUE TITLE
Figma and postman is not tracked?

### ISSUE THREAD

**Issue by AlfinIndrawan** [wakatime/macos-wakatime#74]
so i have two laptops which is personal and work. by installing wakatime i can track the xcode but both figma and postman are not tracked in my machine. so is this my machine problem or wakatime havent ready yet to track those app times? from what i see in leaderboard there is no postman or figma (maybe postman is json but its not even moving at all)

**Comment by alanhamlett** [comment]
Yes, sorry we're working on finding another way to monitor Figma and Postman because they don't trigger `AXFocusedUIElementChanged` and similar accessibility events.

**Comment by alanhamlett** [comment]
Fixed with #150.
